jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferPool.java

```diff
@@ -131,7 +131,7 @@ public void recycle()
         }
     }
 
-    class Bucket
+    public static class Bucket
```

Comment:
Shouldn't this now live in the AbstractByteBufferPool?  it's still an internal implementation, not exposed via the public APIs.